<?xml version="1.0"?>
<launch>

  <!-- launch file parameters -->
  <arg name="topic" default="/a_bot/ur_hardware_interface/script_command"/>
  <arg name="namespace" default="a_bot_gripper"/>
  <arg name="gripper_prefix"	default="a_bot_"/>
  <arg name="config"		default="cmodel_action_controller"/>

  <group ns="$(arg namespace)">
    <!-- CModel TCP Controller -->
    <node name="cmodel_urscript_driver"
	  pkg="aist_robotiq" type="cmodel_urscript_driver.py"
	  output="screen" args="$(arg topic)"/>

    <param name="gripper_prefix" value="$(arg gripper_prefix)"/>
    <rosparam file="$(dirname)/../config/$(arg config).yaml"/>
    <node name="cmodel_action_controller"
	  pkg="aist_robotiq" type="cmodel_action_controller.py"
	  output="screen" />
  </group>

</launch>
